packages/fx-core/package.json

```diff
@@ -60,7 +60,6 @@
     "node-forge": "^0.10.0",
     "node-ts-uuid": "^1.0.8",
     "openapi-types": "^7.2.3",
-    "opn": "^6.0.0",
```

Comment:
do we need to remove the "opn" from "vscode-extension/package.json" either?

Reply:
Yep, removed.